### ISSUE TITLE
Problems after reinstallation of Obsidian and Livesync

### ISSUE THREAD

**Issue by fieldengr2010** [vrtmrz/obsidian-livesync#489]
## Abstract
Reinstallation of Obsidan and Livesync on my Mac has resulted in a lack of pulling sync data down

After reinstalling my Mac from scratch and putting Obsidian and livesync back on and using the encrypted link, I am finding no data comes down to my Mac from the CouchDB server, despite all my folders being visible

## Expected behaviour
- Open Obsidian and a sync check is run
- Nothing is synchronised

## Actually happened
- Sync shows up various files.
- No files synchronised

## Reproducing procedure

1. Configure LiveSync as in the attached material.
2. Click the replication button on the ribbon.
3. Synchronising has begun.
4. About two or three seconds later, scanning of hidden files (wasn't aware of any) and nothing comes down
5. Replication has been stopped. No files synchronised.

Note: If you do not catch the reproducing procedure, please let me know the frequency and signs.

## Report materials
If the information is not available, do not hesitate to report it as it is. You can also of course omit it if you think this is indeed unnecessary. If it is necessary, I will ask you.

### Report from the LiveSync

Report from the inspector/console

```
>Synchronizing hidden files...
plugin:obsidian-livesync:44038 test-2a0d827e50205d62:09/09/2024, 16:29:54->Scanning hidden files.
plugin:obsidian-livesync:44038 test-2a0d827e50205d62:09/09/2024, 16:29:57->Hidden files scanned: 6 files had been modified
plugin:obsidian-livesync:44038 test-2a0d827e50205d62:09/09/2024, 16:29:57->Synchronizing hidden files done
plugin:obsidian-livesync:44038 test-2a0d827e50205d62:09/09/2024, 16:29:58->Scanning hidden files.
plugin:obsidian-livesync:44038 test-2a0d827e50205d62:09/09/2024, 16:30:01->Hidden files scanned: 0 files had been modified
plugin:obsidian-livesync:44038 test-2a0d827e50205d62:09/09/2024, 16:30:01->Before LiveSync, start OneShot once...
plugin:obsidian-livesync:44038 test-2a0d827e50205d62:09/09/2024, 16:30:01->OneShot Sync begin... (pullOnly)
plugin:obsidian-livesync:44038 test-2a0d827e50205d62:09/09/2024, 16:30:01->Remote storage size: 1.25MB
plugin:obsidian-livesync:44038 test-2a0d827e50205d62:09/09/2024, 16:30:01->Replication completed
plugin:obsidian-livesync:44038 test-2a0d827e50205d62:09/09/2024, 16:30:01->LiveSync begin...
plugin:obsidian-livesync:44038 test-2a0d827e50205d62:09/09/2024, 16:30:01->Replication activated
plugin:obsidian-livesync:44038 test-2a0d827e50205d62:09/09/2024, 16:31:01->Scanning hidden files.
plugin:obsidian-livesync:44038 test-2a0d827e50205d62:09/09/2024, 16:31:04->Hidden files scanned: 0 files had been modified
plugin:obsidian-livesync:44038 test-2a0d827e50205d62:09/09/2024, 16:32:01->Scanning hidden files.
plugin:obsidian-livesync:44038 test-2a0d827e50205d62:09/09/2024, 16:32:04->Hidden files scanned: 0 files had been modified
plugin:obsidian-livesync:44038 test-2a0d827e50205d62:09/09/2024, 16:33:01->Scanning hidden files.
plugin:obsidian-livesync:44038 test-2a0d827e50205d62:09/09/2024, 16:33:04->Hidden files scanned: 0 files had been modified
plugin:obsidian-livesync:44038 test-2a0d827e50205d62:09/09/2024, 16:33:41->Log window opened
plugin:obsidian-livesync:44038 test-2a0d827e50205d62:09/09/2024, 16:33:42->Replication is already in progress.
plugin:obsidian-livesync:44038 test-2a0d827e50205d62:09/09/2024, 16:33:43->Chunks saved: doc: .obsidian/workspace.json ,chunks: 1 (new:1, recycled:0, cached:0)
plugin:obsidian-livesync:44038 test-2a0d827e50205d62:09/09/2024, 16:33:43->Replication activated
plugin:obsidian-livesync:44038 test-2a0d827e50205d62:09/09/2024, 16:33:43->STORAGE --> DB:.obsidian/workspace.json: (hidden) Done
plugin:obsidian-livesync:44038 test-2a0d827e50205d62:09/09/2024, 16:34:01->Scanning hidden files.
plugin:obsidian-livesync:44038 test-2a0d827e50205d62:09/09/2024, 16:34:04->Hidden files scanned: 1 files had been modified
plugin:obsidian-livesync:44038 test-2a0d827e50205d62:09/09/2024, 16:35:01->Scanning hidden files.
plugin:obsidian-livesync:44038 test-2a0d827e50205d62:09/09/2024, 16:35:04->Hidden files scanned: 0 files had been modified
```
</details>

### Obsidian debug info
<details>
<summary>Debug info</summary>

```
09/09/2024, 16:29:51->Cache initialized 300 / 250000000000
09/09/2024, 16:29:51->loading plugin
09/09/2024, 16:29:51->Self-hosted LiveSync v0.23.20 0.23.20
09/09/2024, 16:29:51->xxhash for plugin initialised
09/09/2024, 16:29:51->Self-hosted LiveSync has undergone a major upgrade. Please open the setting dialog, and check the information pane.
09/09/2024, 16:29:51->Waiting for ready...
09/09/2024, 16:29:51->Opening Database...
09/09/2024, 16:29:51->Database is now ready.
09/09/2024, 16:29:51->Initialize and checking database files
09/09/2024, 16:29:51->Checking deleted files
09/09/2024, 16:29:51->Collecting local files on the DB: 25
09/09/2024, 16:29:51->Collecting local files on the DB: 50
09/09/2024, 16:29:51->Collecting local files on the DB: 75
09/09/2024, 16:29:51->Collecting local files on the DB: 100
09/09/2024, 16:29:51->Updating database by new files
09/09/2024, 16:29:51->UPDATE DATABASE: Nothing to do
09/09/2024, 16:29:51->UPDATE STORAGE
09/09/2024, 16:29:51->entry not found: _Index_of_Daybook.md
09/09/2024, 16:29:51->entry not found: 20221026-1650 - Egnyte use.md
09/09/2024, 16:29:51->entry not found: 20221014-1035 - Pay.md
09/09/2024, 16:29:51->entry not found: 20221012-1455 - Mtives.md
09/09/2024, 16:29:51->entry not found: 20221014-1051 - Pork.md
09/09/2024, 16:29:51->entry not found: 20220511-1108 - Nt checks.md
09/09/2024, 16:29:51->entry not found: 20221122-1038 - Nilling.md
09/09/2024, 16:29:51->entry not found: 20220808-1019 - R499.md
09/09/2024, 16:29:51->entry not found: 20220811-1001 - Son IP map.md
09/09/2024, 16:29:51->entry not found: 20220810-1545 - planned.md
09/09/2024, 16:29:51->UPDATE STORAGE: DONE:10, FAILED:0, LAST:86
09/09/2024, 16:29:51->entry not found: 20221128-1152 -e.md
09/09/2024, 16:29:51->entry not found: 20221128-1556 - Zabbix 6.2.md
09/09/2024, 16:29:51->entry not found: 20230112-1037 - Snt.md
09/09/2024, 16:29:51->entry not found: 20231204-0856 - RHF.md
09/09/2024, 16:29:51->entry not found: 20230427-0909 - N.md
09/09/2024, 16:29:51->entry not found: 20230426-1646 - BC.md
09/09/2024, 16:29:51->entry not found: 20230213-0958 - Nurobs.md
09/09/2024, 16:29:51->entry not found: 20230314-1541 - Puild.md
09/09/2024, 16:29:51->entry not found: 20230322-0852 - Nail.md
09/09/2024, 16:29:51->entry not found: 20231204-1623 - Ber.md
09/09/2024, 16:29:51->UPDATE STORAGE: DONE:20, FAILED:0, LAST:76
09/09/2024, 16:29:51->entry not found: 20240112-1649 - Zix2.md
09/09/2024, 16:29:51->entry not found: 20240109-1027 - Vaultwn 2.md
09/09/2024, 16:29:52->entry not found: 20231222-1045 - GI024.md
09/09/2024, 16:29:52->entry not found: 20231207-0934 - SAouter.md
09/09/2024, 16:29:52->entry not found: 20240111-0920 - SANB.md
09/09/2024, 16:29:52->entry not found: 20240117-1653 - Oc.md
09/09/2024, 16:29:52->entry not found: 20240109-1133 - NU65 MIG.md
09/09/2024, 16:29:52->entry not found: 365 NCE - Staysociates.md
09/09/2024, 16:29:52->entry not found: 365 NCE - Stay aster.md
09/09/2024, 16:29:52->entry not found: 365 NCE - Stay nchising.md
09/09/2024, 16:29:52->UPDATE STORAGE: DONE:30, FAILED:0, LAST:66
09/09/2024, 16:29:52->entry not found: Daily/Fri 20-01-23.md
09/09/2024, 16:29:52->entry not found: Daily/_Index_of_Daily.md
09/09/2024, 16:29:52->entry not found: From OneNote/2LL.md
09/09/2024, 16:29:52->entry not found: Day Planners/_Index_of_Day Planners.md
09/09/2024, 16:29:52->entry not found: From OneNote/dmin.md
09/09/2024, 16:29:52->entry not found: From OneNote/Appointment System.md
09/09/2024, 16:29:52->entry not found: From OneNote/Dto.md
09/09/2024, 16:29:52->entry not found: 365 NCE - Stay Monthly/_Index_of_365 NCE - Stay Monthly.md
09/09/2024, 16:29:52->entry not found: From OneNotent to MCB.md
09/09/2024, 16:29:52->entry not found: From OneNote/Dark web.md
09/09/2024, 16:29:52->UPDATE STORAGE: DONE:40, FAILED:0, LAST:56
09/09/2024, 16:29:52->entry not found: From OneNote/Delta.md
09/09/2024, 16:29:52->entry not found: From OneNote/md
09/09/2024, 16:29:52->entry not found: From OneNote/F.md
09/09/2024, 16:29:52->entry not found: From OneNote/My others.md
09/09/2024, 16:29:52->entry not found: From OneNote/.md
09/09/2024, 16:29:52->entry not found: From OneNote/.md
09/09/2024, 16:29:52->entry not found: From OneNote/s.md
09/09/2024, 16:29:52->entry not found: From OneNote/Station.md
09/09/2024, 16:29:52->entry not found: From OneNote/ services.md
09/09/2024, 16:29:52->entry not found: From OneNote/_Index_of_From OneNote.md
09/09/2024, 16:29:52->UPDATE STORAGE: DONE:50, FAILED:0, LAST:46
09/09/2024, 16:29:52->entry not found: IP details.md
09/09/2024, 16:29:52->Chunks saved: doc: .obsidian/core-plugins-migration.json ,chunks: 1 (new:0, recycled:1, cached:0)
09/09/2024, 16:29:52->STORAGE --> DB:.obsidian/core-plugins-migration.json: (hidden) Done
09/09/2024, 16:29:52->entry not found: Invoices to raise/_Index_of_Invoices to raise.md
09/09/2024, 16:29:52->entry not found: Once in a blue moon things!/Add to MPC.md
09/09/2024, 16:29:52->entry not found: Once in a blue moon things!/BCS booking form.md
09/09/2024, 16:29:52->entry not found: Office things (Certs, etc)/20221107-1248 -ey.md
09/09/2024, 16:29:52->entry not found: Office things (Certs, etc)/20221103-1637 - AND.md
09/09/2024, 16:29:52->entry not found: Office things (Certs, etc)/20230120-1639 - ining.md
09/09/2024, 16:29:52->entry not found: Office things (Certs, etc)/_Index_of_Office things (Certs, etc).md
09/09/2024, 16:29:52->entry not found: Office things (Certs, etc)/20220927-1219 - Office certifications.md
09/09/2024, 16:29:52->entry not found: Office things (Certs, etc)/20220715-1659 - Things on my to-do list.md
09/09/2024, 16:29:52->UPDATE STORAGE: DONE:60, FAILED:0, LAST:36
09/09/2024, 16:29:52->entry not found: Once in a blue moon things!/Ocovery.md
09/09/2024, 16:29:52->entry not found: Once in a blue moon things!/inute issue.md
09/09/2024, 16:29:52->entry not found: Safe Storage/20220624-1626 - Vault.md
09/09/2024, 16:29:52->entry not found: Safe Storage/20220914-0807 - BC.md
09/09/2024, 16:29:52->entry not found: Safe Storage/BD.md
09/09/2024, 16:29:52->entry not found: Safe Storage/20220817-1138 - .md
09/09/2024, 16:29:52->entry not found: Safe Storage/20221110-1504 - S.md
09/09/2024, 16:29:52->entry not found: Safe Storage/20230206-1036 - S.md
09/09/2024, 16:29:52->entry not found: Safe Storage/20221128-0957 - e.md
09/09/2024, 16:29:52->entry not found: Once in a blue moon things!/_Index_of_Once in a blue moon things!.md
09/09/2024, 16:29:52->UPDATE STORAGE: DONE:70, FAILED:0, LAST:26
09/09/2024, 16:29:52->entry not found: Safe Storage/_Index_of_Safe Storage.md
09/09/2024, 16:29:52->entry not found: Scripts/20221101-1522 - NENT.md
09/09/2024, 16:29:52->entry not found: Scripts/_Index_of_Scripts.md
09/09/2024, 16:29:52->entry not found: Welcome.md
09/09/2024, 16:29:52->entry not found: Scripts/m65.md
09/09/2024, 16:29:52->entry not found: Standard Replies/Booking - 30 minutes.md
09/09/2024, 16:29:52->entry not found: Standard Replies/Remote Agent Load.md
09/09/2024, 16:29:52->entry not found: Standard Replies/cret.md
09/09/2024, 16:29:52->entry not found: Standard Replies/Remote support (MAC).md
09/09/2024, 16:29:52->entry not found: Standard Replies/_Index_of_Standard Replies.md
09/09/2024, 16:29:52->UPDATE STORAGE: DONE:80, FAILED:0, LAST:16
09/09/2024, 16:29:52->entry not found: Z - Filed/2022/04/20220426-1227 - S.md
09/09/2024, 16:29:52->entry not found: Z - Filed/20220429-1648 - H.md
09/09/2024, 16:29:52->entry not found: Z - Filed/20220502-1242 - Owal.md
09/09/2024, 16:29:52->entry not found: Z - Filed/20220429-1659 - LSpace issue.md
09/09/2024, 16:29:52->entry not found: Z - Filed/20220509-1117 - 65.md
09/09/2024, 16:29:52->entry not found: Z - Filed/20220504-1446 - BCumbers.md
09/09/2024, 16:29:52->entry not found: Z - Filed/20220518-1018 - me.md
09/09/2024, 16:29:52->entry not found: Z - Filed/20220513-1637 - tings.md
09/09/2024, 16:29:52->entry not found: Z - Filed/20220519-1618 - Nild.md
09/09/2024, 16:29:52->entry not found: Z - Filed/20220705-1012 - AlmaLinux.md
09/09/2024, 16:29:52->UPDATE STORAGE: DONE:90, FAILED:0, LAST:6
09/09/2024, 16:29:52->entry not found: Z - Filed/20220523-1618 - nvention.md
09/09/2024, 16:29:52->entry not found: Z - Filed/20220816-1610 - on.md
09/09/2024, 16:29:52->entry not found: Z - Filed/20220817-1526 - p.md
09/09/2024, 16:29:52->entry not found: Z - Filed/20220913-1102 - otics.md
09/09/2024, 16:29:52->entry not found: Z - Filed/20221031-1234 - Rg.md
09/09/2024, 16:29:52->entry not found: Welcome.md
09/09/2024, 16:29:52->entry not found: Z - Filed/20221104-1621 - SCAN2GO.md
09/09/2024, 16:29:52->entry not found: Z - Filed/20221101-1125 - N Mac.md
09/09/2024, 16:29:52->entry not found: Z - Filed/20221104-1454 - .md
09/09/2024, 16:29:52->entry not found: Z - Filed/20221202-1623 - DM.md
09/09/2024, 16:29:52->UPDATE STORAGE: DONE:100, FAILED:0, LAST:0
09/09/2024, 16:29:52->entry not found: Z - Filed/220426 - oad.md
09/09/2024, 16:29:52->entry not found: Z - Filed/220426 - ncing.md
09/09/2024, 16:29:52->entry not found: Z - Filed/220426 - Racts.md
09/09/2024, 16:29:52->entry not found: Z - Filed/_Index_of_Z - Filed.md
09/09/2024, 16:29:52->UPDATE STORAGE All done: DONE:106, FAILED:0
09/09/2024, 16:29:52->Chunks saved: doc: .obsidian/workspace.json ,chunks: 1 (new:1, recycled:0, cached:0)
09/09/2024, 16:29:52->STORAGE --> DB:.obsidian/workspace.json: (hidden) Done
09/09/2024, 16:29:54->Initialized, NOW TRACKING!
09/09/2024, 16:29:54->Synchronizing hidden files...
09/09/2024, 16:29:54->Scanning hidden files.
09/09/2024, 16:29:57->Hidden files scanned: 6 files had been modified
09/09/2024, 16:29:57->Synchronizing hidden files done
09/09/2024, 16:29:58->Scanning hidden files.
09/09/2024, 16:30:01->Hidden files scanned: 0 files had been modified
09/09/2024, 16:30:01->Before LiveSync, start OneShot once...
09/09/2024, 16:30:01->OneShot Sync begin... (pullOnly)
09/09/2024, 16:30:01->Remote storage size: 1.25MB
09/09/2024, 16:30:01->Replication completed
09/09/2024, 16:30:01->LiveSync begin...
09/09/2024, 16:30:01->Replication activated
09/09/2024, 16:31:01->Scanning hidden files.
09/09/2024, 16:31:04->Hidden files scanned: 0 files had been modified
09/09/2024, 16:32:01->Scanning hidden files.
09/09/2024, 16:32:04->Hidden files scanned: 0 files had been modified
09/09/2024, 16:33:01->Scanning hidden files.
09/09/2024, 16:33:04->Hidden files scanned: 0 files had been modified
09/09/2024, 16:33:41->Log window opened
09/09/2024, 16:33:42->Replication is already in progress.
09/09/2024, 16:33:43->Chunks saved: doc: .obsidian/workspace.json ,chunks: 1 (new:1, recycled:0, cached:0)
09/09/2024, 16:33:43->Replication activated
09/09/2024, 16:33:43->STORAGE --> DB:.obsidian/workspace.json: (hidden) Done
09/09/2024, 16:34:01->Scanning hidden files.
09/09/2024, 16:34:04->Hidden files scanned: 1 files had been modified
09/09/2024, 16:35:01->Scanning hidden files.
09/09/2024, 16:35:04->Hidden files scanned: 0 files had been modified
09/09/2024, 16:36:01->Scanning hidden files.
09/09/2024, 16:36:04->Hidden files scanned: 0 files had been modified
```
</details>

**Comment by vrtmrz** [comment]
Thank you for opening the issue! And, thanks to your detailed information, we could now the reason. The trouble caused the case-insensitivity handling issue in the previous version.
 
Fortunately, this had been fixed at v0.23.22. Would you mind if I ask you to update the v0.23.22 and check the behaviour, please?

**Comment by fieldengr2010** [comment]
Thanks for coming back to me, I got to the bottom of the problem. I assumed rejoining would pull all the data back down, but I needed to put a copy of the local cache back onto the machine in question. Once in place, all worked normally again.
